Dockerfile

```diff
@@ -39,6 +39,9 @@ RUN wget -q -O /usr/bin/yaml https://github.com/mikefarah/yaml/releases/download
 
 # Build Arguments and environment variables
 ARG EXO_VERSION=5.0.0-RC04
+# Version of chat standalone to install if activated
+ENV EXO_CHAT_STANDALONE_VERSION=1.6.0-RC04
```

Comment:
_STANDALONE_ is not needed here, the variable name should be EXO_CHAT_VERSION.

Even better, is there any way to "guess" the chat version from the already installed chat add-on ?

Reply:
👍 the version can be found on the exo-chat.status. jq was added on the image to easily extract it from the json.